consul/config_templates/consul.yaml

```diff
@@ -575,24 +546,31 @@ spec:
             periodSeconds: 2
             successThreshold: 1
             timeoutSeconds: 5
-
+          resources:
+            limits:
+              cpu: 50m
+              memory: 50Mi
+            requests:
+              cpu: 50m
+              memory: 50Mi
 ---
 # Source: consul/templates/server-statefulset.yaml
 # StatefulSet to run the actual Consul server cluster.
 apiVersion: apps/v1
 kind: StatefulSet
 metadata:
   name: consul-consul-server
-  namespace: consul
+  namespace: {{.namespace}}
   labels:
     app: consul
     chart: consul-helm
-    heritage: Tiller
+    heritage: Helm
     release: consul
+    component: server
 spec:
   serviceName: consul-consul-server
   podManagementPolicy: Parallel
-  replicas: 3
+  replicas: 1
```

Comment:
Should we add `replicas` into the `value.yaml` return the choice to the end-user?

Reply:
`replicas`  is already in the `consul-values.yaml` file (`server.replicas`)
I agree that, eventually, the choice about the number of replicas (and other parameters) should be the given to the end-user. This should be solved in other issues. 

This PR fixes #2 (update consul) and #1 (support 1 node), and this was discussed briefly and agreed on at the Meshery Development Meeting on 12.8.2020.

Reply:
I'm not quite fluent in emoji, but I assume it means ok here and that I can resolve the conversation :-)